src/main/java/no/nav/familie/integrasjoner/kodeverk/KodeverkController.kt

```diff
@@ -0,0 +1,16 @@
+package no.nav.familie.integrasjoner.kodeverk
+
+import no.nav.security.token.support.core.api.Unprotected
+import org.springframework.http.MediaType
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@Unprotected
+@RequestMapping(path = ["/api/kodeverk/poststed"], produces = [MediaType.TEXT_PLAIN_VALUE])
+class PostnummerController(val kodeverkService: KodeverkService) {
+
+    @GetMapping("/{postnummer}")
+    fun hentPoststedFor(@PathVariable postnummer: String): String {
+        return kodeverkService.hentPoststedFor(postnummer)
```

Comment:
Har dere vurdert å bruke ressurs her? Jeg mener å huske at alle andre endepunkter nå gjør det.

Reply:
ah stemmer, skal titte på det! Takk for feedback :)